brokerapi/brokers/gcp_service_broker.go

```diff
@@ -589,6 +598,13 @@ func InitCatalogFromEnv() ([]models.Service, error) {
 	}
 	servicePlans[bigtableServiceId] = append(servicePlans[bigtableServiceId], bigtablePlans...)
 
+	// set up bigtable custom plans
+	spannerPlans, spannerServiceId, err := getDynamicPlans("SPANNER_CUSTOM_PLANS", spanner.MapPlan)
```

Comment:
absolutely agree that this bit should be refactored, but I want to do it as its own PR.